src/components/EditCollapse.js

```diff
@@ -82,6 +82,8 @@ const EditCollapse = ({ ...props }) => {
   const removals = changes.filter((change) => change.remove || change.replace).length;
   const newTotal = cube.cards.length + additions - removals;
 
+  const viewParams = new URLSearchParams(window.location.search).get('view');
```

Comment:
The `CubeListNavbar` component, from which this component is called, has access to a `cubeView` property which contains this data already. I would suggest instead of querying for it here again, edit `CubeListNavbar` so that it passes that property into `EditCollapse` and then just use the passed prop.